src/common/ui-components/Slide/Slide.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+
+import styles from './Slide.module.scss';
+
+export interface SlideProps {
+  children: React.ReactNode;
+  backgroundImage: any;
+}
+
+const Slide = (props: SlideProps) => {
+  return (
+    <div
+      className={classNames(styles.container)}
+      style={{
+        backgroundImage: props.backgroundImage
+          ? `url(${props.backgroundImage})`
+          : 'url("https://images.unsplash.com/photo-1557261651-f893a96f357e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80")',
```

Comment:
> url("https://images.unsplash.com/photo-1557261651-f893a96f357e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80")

Please provide the fallback url as an optional prop with a default value that points to some file in our own repository.

Reply:
done